app/tests.py

```diff
@@ -26,7 +26,7 @@
     ContractFactory,
     InstallmentConditionFactory,
     InstallmentFactory,
-)
+    EventFactory)
```

Comment:
Nit put that brace in a new line. You can define this kind of automatization in this settings https://www.jetbrains.com/help/pycharm-edu/configuring-code-style.html